docs/src/constraints.md

```diff
@@ -257,6 +258,35 @@ julia> dual_start_value(con)
 2.0
 ```
 
+A vector constraint will require a vector warmstart:
+
+```jldoctest constraint_dual_start_vector; setup=:(model=Model())
+julia> @variable(model, x)
+x
+
+julia> @constraint(model, con, [x, x] in SecondOrderCone())
+con : [x, x] in MathOptInterface.SecondOrderCone(2)
+
+julia> dual_start_value(con)
+
+julia> set_dual_start_value(con, [1.0, 3.0])
+
+julia> dual_start_value(con)
+2-element Array{Float64,1}:
+ 1.0
+ 3.0
+```
+
+To set the previous solution to a new solve, use:
```

Comment:
"To take the dual solution from the last solve and use it as the starting point for a new solve, use:"